src/excmds.ts

```diff
@@ -2877,6 +2897,26 @@ async function getclip() {
     }
 }
 
+/**
+ * Fetches primary selection only
+ *
+ * @hidden
+ */
+//#background_helper
+async function getprimsel() {
```

Comment:
At first I had made getclip/setclip parametric so you could decide exactly what you wanted. Then we removed that because nothing used it yet. Maybe we could make them parametric again to reduce code duplication?